editor/project_manager.cpp

```diff
@@ -1123,88 +1211,163 @@ ProjectManager::ProjectManager() {
 
 			open_btn = memnew(Button);
 			open_btn->set_text(TTR("Edit"));
 			open_btn->set_shortcut(ED_SHORTCUT("project_manager/edit_project", TTR("Edit Project"), KeyModifierMask::CMD_OR_CTRL | Key::E));
 			open_btn->connect("pressed", callable_mp(this, &ProjectManager::_open_selected_projects_ask));
-			tree_vb->add_child(open_btn);
+			project_list_sidebar->add_child(open_btn);
 
 			run_btn = memnew(Button);
 			run_btn->set_text(TTR("Run"));
 			run_btn->set_shortcut(ED_SHORTCUT("project_manager/run_project", TTR("Run Project"), KeyModifierMask::CMD_OR_CTRL | Key::R));
 			run_btn->connect("pressed", callable_mp(this, &ProjectManager::_run_project));
-			tree_vb->add_child(run_btn);
+			project_list_sidebar->add_child(run_btn);
 
 			rename_btn = memnew(Button);
 			rename_btn->set_text(TTR("Rename"));
 			// The F2 shortcut isn't overridden with Enter on macOS as Enter is already used to edit a project.
 			rename_btn->set_shortcut(ED_SHORTCUT("project_manager/rename_project", TTR("Rename Project"), Key::F2));
 			rename_btn->connect("pressed", callable_mp(this, &ProjectManager::_rename_project));
-			tree_vb->add_child(rename_btn);
+			project_list_sidebar->add_child(rename_btn);
 
 			manage_tags_btn = memnew(Button);
 			manage_tags_btn->set_text(TTR("Manage Tags"));
-			tree_vb->add_child(manage_tags_btn);
+			project_list_sidebar->add_child(manage_tags_btn);
 
 			erase_btn = memnew(Button);
 			erase_btn->set_text(TTR("Remove"));
 			erase_btn->set_shortcut(ED_SHORTCUT("project_manager/remove_project", TTR("Remove Project"), Key::KEY_DELETE));
 			erase_btn->connect("pressed", callable_mp(this, &ProjectManager::_erase_project));
-			tree_vb->add_child(erase_btn);
+			project_list_sidebar->add_child(erase_btn);
+
+			Control *filler = memnew(Control);
+			filler->set_v_size_flags(Control::SIZE_EXPAND_FILL);
+			project_list_sidebar->add_child(filler);
 
 			erase_missing_btn = memnew(Button);
 			erase_missing_btn->set_text(TTR("Remove Missing"));
 			erase_missing_btn->connect("pressed", callable_mp(this, &ProjectManager::_erase_missing_projects));
-			tree_vb->add_child(erase_missing_btn);
-
-			tree_vb->add_spacer();
-
-			about_btn = memnew(Button);
-			about_btn->set_text(TTR("About"));
-			about_btn->connect("pressed", callable_mp(this, &ProjectManager::_show_about));
-			tree_vb->add_child(about_btn);
+			project_list_sidebar->add_child(erase_missing_btn);
 		}
 	}
 
+	// Asset library view.
 	if (AssetLibraryEditorPlugin::is_available()) {
 		asset_library = memnew(EditorAssetLibrary(true));
-		asset_library->set_name(TTR("Asset Library Projects"));
-		tabs->add_child(asset_library);
+		asset_library->set_name("AssetLibraryTab");
+		_add_main_view(MAIN_VIEW_ASSETLIB, TTR("Asset Library"), Ref<Texture2D>(), asset_library);
 		asset_library->connect("install_asset", callable_mp(this, &ProjectManager::_install_project));
 	} else {
 		print_verbose("Asset Library not available (due to using Web editor, or SSL support disabled).");
```

Comment:
Wouldn't it be better to make a tooltip like in news?

Reply:
Yeah, I'm in favor. I didn't want to change the status quo here, as I didn't exactly know why it was done like that. But since we have the disabled state we might just as well use it.